### DISCUSSION TITLE
How to donate?

### DISCUSSION THREAD

**Discussion by acmpo6ou** [gsantner/markor#2348] [Q&A] [answered]
@gsantner Hello! I love Markor, and I want to support you, I found this site, is this yours? https://gsantner.net/page/supportme.html

If yes, I don't see an option to donate. Is this something you'd like to add? Also would be nice to add it to the README.

I want to donate as a thanks for making Markor for us for free and open source.

**Comment by gsantner** [comment, marked as answer]
Please pick a charity (i.e. health section?). Living longer and more happier benefits not only me but everybody. Just like Markor.

Thanks.

**Reply by acmpo6ou** [reply]
Oh my god @gsantner, You are one of the nicest people out there. Not only do You gift all this for us for free, but you don't even accept donations. I only wish that everyone be as nice to You as You are to them ❤️ And I'm especially sorry for all the stuff You have to go through as an open source maintainer.

**Reply by RockyJE** [reply]
I ditto @acmpo6ou's sentiments.
Markor has been my **number 1 essential app that use Dozens of times every day**, and it just keeps getting better!
I often donate $$ to developers of apps that I use, and since I can't do that I'll donate to a local charity that supports women and children; gods knows they really need it.

Thank you @gsantner for this wonderful app. 🏆
